### PR TITLE
skeleton django app

### PR DESCRIPTION
**Pull request by cliffxuan**
skeleton

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+=====================
+MOJ Product Dashboard
+=====================
+
```
Comment (s-block) [inline README.rst:4]: Can you put the code climate badges in to the readme - https://codeclimate.com/github/ministryofjustice/moj-product-dashboard

Comment (cliffxuan) [inline README.rst:4]: sure. will do.

File: requirements/production.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+-r testing.txt
```
Comment (s-block) [inline requirements/production.txt]: can probably delete this and use base for prod for the necessary packages to run the app. then use test/local/docs etc to install the extras

**Comment by s-block** [comment]
other than the above looks good to merge 👍

**Comment by cliffxuan** [comment]
addressed the issues in the comments. going to merge it.

**Pull request merged**